google-cloud-apis/java-apis/resources/META-INF/cloud-apis-java.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ Copyright 2018 Google Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<idea-plugin>
+    <depends>com.intellij.modules.java</depends>
+
+    <extensions defaultExtensionNs="com.google.gct.cloudapis">
+        <cloudApiRunConfiguration
+                implementation="com.google.cloudtools.intellij.cloudapis.java.JavaApplicationCloudApiRunConfigurationProvider"/>
```

Comment:
is this the only class that we'll have as part of the cloud-apis-java module?

Reply:
what I mean is, does everything thats now in google-cloud-apis either belong there or will be moved to the maven sub-module later?

Reply:
Everything Maven related belongs to maven module. This is the only java non-maven dependency - this will allow us to use plain Java configurations without Maven installed.